services/server/src/index.ts

```diff
@@ -79,6 +79,7 @@ app.use(
   express.static(serverConfig.assets.directory.receipts)
 );
 app.use("/api/v1/auth", routes.auth);
+app.use("/api/v1/contactRequest", routes.contactRequest);
```

Comment:
We use the dash notation in the api routes. Simliar to what you did in the availability slots route. 

```suggestion
app.use("/api/v1/contact-request", routes.contactRequest);
```